cli/cmd/generate_aws.go

```diff
@@ -117,7 +117,7 @@ This command can also be run in noninteractive mode. See help output for more de
 			}
 
 			// Write-out generated code to location specified
-			dirname, location, err := writeGeneratedCodeToLocation(cmd, hcl)
+			dirname, location, err := writeGeneratedCodeToLocation(cmd, hcl, "aws")
```

Comment:
Wondering if it would be a good pattern to have the file output name declared as a const at the top of each generate_something file? No strong feelings on this, just a thought.

Reply:
I had considered that. Didn't feel like making it a CONST gave us much so decided to go with this approach. I have no strong feelings on this either, so happy enough to change if others feel strongly about it.